runtime/makelib/mkconstants.mk.ftl

```diff
@@ -24,10 +24,12 @@
 </#list>
 
 # Define the Java Version we are compiling
+# This VERSION_MAJOR variable will be overridden by command line option 
+# when there is one such as VERSION_MAJOR=xx.
```

Comment:
I don't think `VERSION_MAJOR` will always be defined on the command line (in all recursive calls to make), so the definition needs to use `?=`, not `:=`.

Reply:
I have been convinced that `:=` is fine (perhaps even preferred).